espnet2/enh/decoder/stft_decoder.py

```diff
@@ -23,6 +23,9 @@ def __init__(
         normalized: bool = False,
         onesided: bool = True,
         default_fs: int = 16000,
+        spec_transform_type: str = None,
+        spec_factor: float = 0.15,
+        spec_abs_exponent: float = 0.5,
```

Comment:
It is better to add a docstring to clarify the functions of new arguments.

Reply:
Done.